Models/GenderEnum.cs

```diff
@@ -0,0 +1,8 @@
+namespace AppBackendTraining.Models
+{
+    public enum GenderEnum
+    {
+        m,
```

Comment:
Pascal convention for enum members, see https://docs.microsoft.com/en-us/dotnet/standard/design-guidelines/capitalization-conventions